### PR TITLE
[READY] [MDB IGNORE] Excising the fat: Splicing areas.dmi into multiple files, also some minor repathing

### PR DESCRIPTION
**Pull request by Jolly-66**
## About The Pull Request
Areas.dmi right now houses *all* of our mapped turfs icons (which is roughly 400 icons). Not an issue, but it's *incredibly* large and clunky to navigate right now. This isn't an issue for the average coder and/or player code diving, but it is for mappers wanting to add new turfs. Currently, the file has *some* organization, but its still an overall mess. This PR aims to slice the behemoth with multiple .dmi files corresponding to specific areas, such as the following:
- Station related areas
- Cent Comm-Z related turfs (wiz ship, syndie base, ect)
- Ruins for space and Lavaland*
- Away missions/gateways 
- Misc
*Lavaland may not be its own file, instead, I may opt to just condense space ruins into one file.

I also plan to repath `/area/*` -> `/area/station/*` for **station turf** only. This is to clean it up, as most other turfs follow this format (that being `/area/turf_zone/*`). 

I'm also writing an update paths file as I go along.

Jollys shit to test because it will break

- [x] Convert *all* station turfs to `/area/station/*`
- [x] Finish the update paths txt file
- [x] Ensure type paths are updated correctly for Stations
- [ ] Ensure icons for "Less" important maps are there (Shuttles, ruins, ect)
- [x] Ensure icons for Station maps are there
- [x] Make sure anything using old station turf type paths are properly updated
- [ ] Add unused/delete turf icons to the archive depot, if the spritetainers so desire

## Why It's Good For The Game
None of this player facing, but the bugs arising from it may be. Consistency, ease of access for mappers and coders alike is much appreciated. I don't know what to put for the CL explicitly, so yell at me about that.

## Changelog

:cl: Jolly
code: The typepath used for station turfs has been changed from /area/* to /area/station/* for clarity and consistency. 
code: Space_Station_13_areas.dm had its station specific turf moved to station.dm
imageadd: Some turfs were given icons (which didn't exist prior).
imagedel: Most unused/odd ball turfs were removed (like dk_yellow).
/:cl:

### CODE REVIEW

File: code/modules/awaymissions/cordon.dm [hunk 1]
```diff
@@ -45,6 +45,7 @@
 
 /area/cordon
 	name = "CORDON"
+	icon = 'icons/turf/areas/areas_station.dmi'
```
Comment (Fikou) [inline code/modules/awaymissions/cordon.dm:48]: cordons arent for the station

**Review by Fikou — CHANGES_REQUESTED** [review]
areas being in the turf folder made sense when it was just 1 file. theres a bunch now so i think it should just be a new subfolder in icons

**Comment by Jolly-66** [comment]
> areas being in the turf folder made sense when it was just 1 file. theres a bunch now so i think it should just be a new subfolder in icons

Are you suggesting that instead of what it is now (/icons/areas/turf/)
It should probably be like `/icons/turf`?

**Comment by Jolly-66** [comment]
For the most part, everything is fine.
Ruins turf used those old random colored turf icons, its not an inherent issue now (unless we really want something that isn't a checkerboard), I'm saying this PR is ready to go.

**Comment by Jolly-66** [comment]
Fucking maps augh

**Comment by Jolly-66** [comment]
Icebox is breaking this PR I have 0 fucking idea whats going on

The codes fine here
The codes fine on my local
The map is fine in the map editor

**Review by Imaginos16 — APPROVED** [review]
I have no idea why a spritetainer needs to give approval on a mass rework of area sprites which are things that don't touch visual aesthetics of the game, but here it goes.

By god you're absolutely deranged for pulling this off.

**Comment by Fikou** [comment]
> > areas being in the turf folder made sense when it was just 1 file. theres a bunch now so i think it should just be a new subfolder in icons
> 
> Are you suggesting that instead of what it is now `/icons/areas/turf/` It should probably be like `/icons/turf`?

yes

File: code/modules/surgery/bodyparts/_bodyparts.dm [hunk 2]
```diff
@@ -39,8 +39,6 @@
 
 	/// BODY_ZONE_CHEST, BODY_ZONE_L_ARM, etc , used for def_zone
 	var/body_zone
-	/// The body zone of this part in english ("chest", "left arm", etc) without the species attached to it
-	var/plaintext_zone
```
Comment (Kapu1178) [inline code/modules/surgery/bodyparts/_bodyparts.dm:43]: ?????????????

**Review by Kapu1178 — CHANGES_REQUESTED** [review]

File: code/modules/surgery/bodyparts/_bodyparts.dm (hunk 2 above)
Comment (san7890) [reply code/modules/surgery/bodyparts/_bodyparts.dm:43]: we're having some conflict problems

**Comment by Jolly-66** [comment]
This is a goddamn nightmare

**Comment by Jolly-66** [comment]
Ok you know what I'll just fucking redo this when I'm done my finals

**Pull request closed without merging**